tests/integration/test_charm.py

```diff
@@ -107,13 +107,14 @@ async def test_build_and_deploy(ops_test: OpsTest, local_charm: Path) -> None:
     await integrate_dependencies(ops_test)
 
     await ops_test.model.wait_for_idle(
-        apps=[HYDRA_APP, DB_APP, PUBLIC_INGRESS_APP, INTERNAL_INGRESS_APP],
+        apps=[HYDRA_APP, DB_APP, PUBLIC_INGRESS_APP, ADMIN_INGRESS_APP],
         raise_on_blocked=False,
         status="active",
         timeout=5 * 60,
     )
 
 
+@pytest.mark.skip
```

Comment:
What is the reason to skip these tests?

Reply:
ah, forgot to remove those when I did debugging. Let me remove those.